File: Drivers/FSCC Drivers/linux/lib/python/examples/memory-cap.py
import fscc

if __name__ == '__main__':
    p = fscc.Port(0)

    p.memory_cap.input = 1000000  # 1 MB
    p.memory_cap.output = 2000000  # 2 MB

    input = p.memory_cap.input
    output = p.memory_cap.output

File: Drivers/FSCC Drivers/linux/examples/memory-cap.c
#include <fcntl.h> /* open, O_RDWR */
#include <unistd.h> /* close */
#include <fscc.h> /* FSCC_* */

int main(void)
{
    int fd = 0;
    struct fscc_memory_cap memcap;

    fd = open("/dev/fscc0", O_RDWR);

    memcap.input = 1000000; /* 1 MB */
    memcap.output = 1000000; /* 1 MB */

    ioctl(fd, FSCC_SET_MEMORY_CAP, &memcap);

    close(fd);

    return 0;
}

